### PR TITLE
Translation/el gr

### PR DESCRIPTION
**Pull request by athanasiosem**
We are excited to announce that after months of efforts the Greek translation of this project is now ready.
We are happy we are contributing to this project and we would appreciate your feedback.

### CODE REVIEW

**Comment by athanasiosem** [comment]
Hello @PauloASilva.

The changes that are required have been made. 
Please let us know if there is anything else we need to change.

Best Regards,
Athanasios Emmanouilidis

File: 2019/el-gr/src/0xd1-acknowledgments.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* philippederyck
+* pleothaud
+* r00ter
+* Raj kumar
+* Sagar Popat
+* Stephen Gates
+* thomaskonrad
+* xycloops123
+* Athanasios Emmanouilidis
+* Apostolos Giannakidis
```
Comment (PauloASilva) [inline 2019/el-gr/src/0xd1-acknowledgments.md:42]: This list should remain synchronized with the master version as well as other translations.
The names in this list correspond to those that helped putting together the 2019 version.

Translators are acknowledge [here](https://owasp.org/www-project-api-security/#div-translations).
Additionally, you can add a "Translation Notes" sub-section in the "Release Notes" section as [the one you find in the Portuguese translation](https://github.com/OWASP/API-Security/blob/master/2019/pt-pt/src/0x04-release-notes.md#notas-da-tradu%C3%A7%C3%A3o).

**Review by PauloASilva — CHANGES_REQUESTED** [review]
Would you mind addressing [my comment regarding the Acknowledgements][1]?
Everything else looks fine and ready for merge.

Please kindly provide the names and corresponding social links (e.g. twitter, linkedin), you want to acknowledge for this translation.

[1]: https://github.com/OWASP/API-Security/pull/73/files#r1147537375

**Comment by PauloASilva** [comment]
Hi Folks,
Amazing job you've done here!

The [Greek translation is now published][1] and the [translations list updated accordingly][2].

Cheers,
Paulo A. Silva

[1]: https://github.com/OWASP/API-Security/tree/master/2019/el-gr/src
[2]: https://owasp.org/www-project-api-security/#div-translations

**Pull request closed without merging**